### DISCUSSION TITLE
New sell system

### DISCUSSION THREAD

**Discussion by ghost** [gschurck/tradoge#13] [Ideas]
Hey ,

Is it possible to do something if I buy it at X $ and it goes Y% lower it well sell directly and not wait for the time asked?

Thank you

**Comment by gschurck** [comment]
Hello,
Good idea, it's like a stop loss if it is a bad new. It means the bot has to track the price frequently during this time. I will probably add it to the roadmap, thanks.

**Reply by ghost** [reply]
It can just follow the price each 5-10sec after buying to see if its under X% of the price I bought

**Reply by gschurck** [reply]
Absolutely.
